### DISCUSSION TITLE
Keep receiving error when trying to use refiner

### DISCUSSION THREAD

**Discussion by WulfByteSD** [lllyasviel/Fooocus#3072] [Q&A]
I might not be understanding how to use the refiner properly, but i keep receiving this error message when i try it, doesnt matter what i choose, it throws the error message
`2024-06-01 15:58:20 app-1  | /content/entrypoint.sh: line 33:    16 Killed                  python launch.py $*`

If it helps, im using foocus through Docker

**Comment by mashb1t** [comment]
May i ask what your system specs are? Please ensure you fulfill the minimum specs described in the readme.
You're probably also not able to run Fooocus without docker as either swap has not been set up correctly or your GPU has some issues.
See 
https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md#killed

**Reply by WulfByteSD** [reply]
I have a Nvidia 3060 gpu, and 32gbs (4x8gb) of ram, according to the docs though, system swap shouldn't need to be setup, considering windows does this for me, plus I have ample space on my ssd.. I was hoping docker would keep from having such issues since the way containerization works. But it seems not likely.. any useful tips?

**Reply by mashb1t** [reply]
That's true, Windows should recognize this automatically, I'm also using automated page size. Can you please check if you can start run Fooocus without Docker (download + extract the zip file, the run the run.bat) just to narrow down the issue?
Plus check where your current page file is located (same view where you see that it's automated), as some users already experienced issues in the past when these were not on the same drive.

**Reply by WulfByteSD** [reply]
Ok, took me a minute to get everything together, but i finally just downloaded the windows file and set it up, so its not throwing the error outside the docker container, but it is inside, it makes me wonder if there's an issue setting up python inside the container

**Reply by mashb1t** [reply]
one more idea, maybe too late: the shell script should use LF line endings, but you might have edited it or your editor might have changed it to CRLF. Please check, i've added this already to .gitattributes, but you might have overridden this in your global git config.

**Reply by peoplelikethesun** [reply]
When using docker to start fooocus, please pay attention to your memory and disk ssd. My computer has 16g memory and rxt4060, but when running fooocus, the memory and ssd read and write time are almost full. When I turn off some other software, it runs normally. You can try

**Comment by atkaimage** [comment]
Hi.
Were you able to solve this problem in the end?
I have run into the same error message and had no luck with troubleshooting.
Checked minimum requirements and I am good. Checked system swap and it is also good.
I am not very good in coding so I was not able to test the other advised solutions,
